src/js/view/controls/components/timeslider.js

```diff
@@ -298,6 +305,7 @@ class TimeSlider extends Slider {
         } else {
             ariaText = `${timeFormat(position)} of ${timeFormat(duration)}`;
         }
+        this.timeUpdateKeeper.innerHTML = ariaText;
```

Comment:
Use `textContent` instead of `innerHTML` when there is no HTML that needs to be parsed. This is a plain time string - no HTML.